pkg/testing/cli/harness.go

```diff
@@ -76,6 +76,14 @@ func CreateExitHandlerFunc(t *testing.T, tee *Tee, expectedExitCode int, expecte
 	return exitHandler
 }
 
+func PreventExitHandlerFunc(t *testing.T) func(int) bool {
```

Comment:
So what's happening in some tests is, that if they are setup wrong and the code actually exits (using an atexit method for example), the program/test execution will stop directly. This means that no assertions will be run marking the test effectively as passed. Even worse, subsequent tests won't run. We had this in other test as well. 

Using this handler an os.Exit will be converted to a panic. This means the test will actually fail (with a panic, but at least it does not get unnoticed). 

I will try to find better name and add some godocs. Hope it becomes clearer then